dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/enums/StockIndex.kt

```diff
@@ -12,6 +12,6 @@ enum class StockIndex {
     PrimeStandards,
     Sdax,
     TecDax,
-    ScaleHdax,
+    Hdax,
```

Comment:
Generelle Findings:
[ ] Flackernder Search-Header
[ ] Pagination nicht getestet

Reply:
Da tooltips und datatables von component library genommen wurden, haben wir tests für die richtige Konfiguration geschrieben.

Reply:
flackernder header ist noch da

Reply:
fixed for chrome and firefox

Reply:
jup - jetzt gehts. danke!